test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -77,7 +77,7 @@ abstract class LorisIntegrationTest extends TestCase
             $database['host'],
             1
         );
-        $this->url = $this->config->getSetting("url");
+        $this->url = $this->factory->settings()->getBaseURL();
```

Comment:
You need to try "host" instead of getBaseURL.
BaseURL has been removed by https://github.com/aces/Loris/pull/6019.

Reply:
Only IP address could work in this case. One way is to switch back to the old 'url' mode for the purpose of the test (with DB data for url), one way is to use IP address directly which might be very hard to do. Also @driusan @ridz1208

Reply:
In this case, integration test doesn't need a IP address.

Reply:
Yes, it works, https://travis-ci.org/github/aces/Loris/jobs/689632710

Reply:
At least for now all works fine, waiting for the result.

Reply:
https://travis-ci.org/github/aces/Loris/jobs/689632710 has passed, I have put the fix in the PR #6555 if you need.

Reply:
The fix in PR #6555 is simply adding the "URL" setting back in the Config table. I'm not sure if this is a good fix, since the "URL" was specifically removed in PR #6019 and #6549

Reply:
I will do another trial, I reverted those SQL because the SQL doesn't pose much problem.

Reply:
It seems that changing url directly will work, pending the Travis to finish

Reply:
#6555 has passed the Travis with the method from @kongtiaowang , though Github displays a yellow light.